module2/projects/bulk_discounts.md

```diff
@@ -0,0 +1,143 @@
+# Bulk Discounts
+
+This project is an extension of the Little Esty Shop group project. You will add functionality for merchants to create bulk discounts for their items. A "bulk discount" is a discount based on the quantity of items the customer is buying, for example "20% off orders of 10 or more items".
+
+## Learning Goals
+
+* Write migrations to create tables and relationships between tables
+* Implement CRUD functionality for a resource using forms (form_tag or form_with), buttons, and links
+* Use MVC to organize code effectively, limiting the amount of logic included in views and controllers
+* Use built-in ActiveRecord methods to join multiple tables of data, make calculations, and group data based on one or more attributes
+* Write model tests that fully cover the data logic of the application
+* Write feature tests that fully cover the functionality of the application
+
+## Bulk Discounts
+
+Bulk Discounts are subject to the following criteria:
+
+* Bulk discounts should have a percentage discount as well as a quantity threshold
+* Bulk discounts should belong to a Merchant
+* A Bulk discount is eligible for all items that the merchant sells. Bulk discounts for one merchant should not affect items sold by another merchant
```

Comment:
This line in conjunction with line 24 has always confused me a bit about this project. So a bulk discount belongs to a merchant, but the threshold is on a per item basis. So, I wouldn't have a bulk discount for a quantity of 10 different items combined that a merchant sells, but rather I would have a bulk discount that affects only one item at a time? 
So assuming these items all belong to the same merchant:
```
Item 1 x3
Item 2 x 5
Item 3 x 2
No bulk discount for 10 items applied
```
But here: 
```
Item 1 x 10
Bulk discount for 10 items applied
```

Reply:
Lol I wrote all this out before reading line 25. I understand.